.mergify.yml

```diff
@@ -27,3 +27,13 @@ pull_request_rules:
       merge:
         method: squash
         strict: smart
+  - name: String Import - Auto Merge
+    conditions:
+      - author=github-actions[bot]
+      - title~=string-import*
+      - files~=^Blockzilla\/[^\/]*\/[^\/]*.lproj\/*
+      - files~=^Blockzilla\/[^\/]*.lproj\/*
+    actions:
+      merge:
+        method: rebase
+        strict: smart
```

Comment:
Mergify is dropping support for `strict: smart` - I know we use it in the other workflows too, but maybe this is a good moment to replace it with what they recommend. Or should we do this as a new issue and just change that in all workflows?

See https://blog.mergify.com/strict-mode-deprecation/

Reply:
oh that's right... @st3fan we can try replacing it here and when we see it works change the other workflows?